db/migrate/20220718221920_change_clumn_active_to_status_on_groups.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeClumnActiveToStatusOnGroups < ActiveRecord::Migration[6.0]
```

Comment:
We have a typo here at the class name:

```suggestion
class ChangeColumnActiveToStatusOnGroups < ActiveRecord::Migration[6.0]
```

And also the filename.